plugins/modules/system/sysrc.py

```diff
@@ -0,0 +1,277 @@
+
+    def destroy(self):
+        if self.module.check_mode:
+            self.changed = True
+            return
+
+        (rc, out, err) = self.run_sysrc('-x', self.name)
+        if self.has_unknown_variable(out, err):
+            return False
+
+        self.changed = True
+        return True
+
+    def value_present(self):
+        if self.module.check_mode:
+            self.changed = True
+            return
+
+        setstring = '%s+=%s%s' % (self.name, self.delim, self.value)
+        (rc, out, err) = self.run_sysrc(setstring)
+        if out.find("%s:" % self.name) == 0:
+            values = out.split(' -> ')[1].strip().split(self.delim)
+            if self.value in values:
+                self.changed = True
+                return True
+            else:
+                return False
+        else:
+            return False
+
+    def value_absent(self):
+        if self.module.check_mode:
+            self.changed = True
+            return
+
+        setstring = '%s-=%s%s' % (self.name, self.delim, self.value)
+        (rc, out, err) = self.run_sysrc(setstring)
+        if out.find("%s:" % self.name) == 0:
+            values = out.split(' -> ')[1].strip().split(self.delim)
+            if self.value in values:
+                return False
+            else:
+                self.changed = True
+                return True
+        else:
+            return False
+
+    def run_sysrc(self, *args):
+        cmd = [self.sysrc, '-f', self.path]
+        if self.jail:
+            cmd += ['-j', self.jail]
+        cmd.extend(args)
+
+        (rc, out, err) = self.module.run_command(cmd)
+        if self.module._verbosity > 0:
+            self.cmd = cmd
+
+        if self.module._verbosity >= 4:
+            self.cmd_output = out
+
+        return (rc, out, err)
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            name=dict(type='str', required=True),
+            value=dict(type='str', default=None),
+            state=dict(type='str', default='present', choices=['absent', 'present', 'value_present', 'value_absent']),
+            path=dict(type='str', default='/etc/rc.conf'),
+            delim=dict(type='str', default=' '),
+            jail=dict(type='str', default=None),
+        ),
+        supports_check_mode=True,
+    )
+
+    name = module.params.pop('name')
+    # OID style names are not supported
+    if not re.match('^[a-zA-Z0-9_]+$', name):
+        module.fail_json(
+            msg="Name may only contain alpha-numeric and underscore characters"
+        )
+
+    value = module.params.pop('value')
+    state = module.params.pop('state')
+    path = module.params.pop('path')
+    delim = module.params.pop('delim')
+    jail = module.params.pop('jail')
+    result = dict(
+        name=name,
+        state=state,
+        value=value,
+        path=path,
+        delim=delim,
+        jail=jail
+    )
+
+    rcValue = sysrc(module, name, value, path, delim, jail)
```

Comment:
Python follows in general snake_case.
```suggestion
    rc_value = sysrc(module, name, value, path, delim, jail)
```

Reply:
This has been changed from `rcValue` to `rc_value`